app/assets/javascripts/app/hms-formatter.js

```diff
@@ -0,0 +1,13 @@
+var root = window.LoginGov || {};
+
+root.hmsFormatter = function(milliseconds) {
+  var seconds = milliseconds / 1000;
+  var minutes = parseInt(seconds / 60, 10);
+  var remainingSeconds = parseInt(seconds % 60, 10);
+
+  var displayMinutes = minutes == 0 ? '' :
+    minutes + ' minute' + (minutes !== 1 ? 's' : '') + ' and ';
```

Comment:
I see this didn't really change in the diff, but I'm now seeing this is not i18n-friendly. I'll make an issue to track i18n-ing this